dbfit-java/sqlserver/src/integration-test/resources/001_create_acceptance_test_objects_sqlserver.sql

```diff
@@ -0,0 +1,231 @@
+USE FitNesseTestDB

```

Comment:
This file seems to have window-style end of lines. Now that we're on Linux - would it be possible to convert it to Unix style (dos2unix)?